test/example-types/terms_stats.js

```diff
@@ -39,6 +39,38 @@ describe('term_stats', () => {
       ],
       ...x
     )
+  let includeTest = (...x) =>
+    sequentialResultTest(
+      [
+        {
+          aggregations: {
+            twoLevelFilter: {
+              twoLevelAgg: {
+                buckets: [
+                  {
+                    key: 'City of Deerfield',
+                    doc_count: 50,
+                    twoLevelAgg: {
```

Comment:
pretty sure this is not what would be actually returned since I don't think we generate `twoLevelAgg` anymore under any case as the inner agg